templates/store/search.html

```diff
@@ -40,16 +40,14 @@
     {% endif %}
   {%- endmacro %}
 
-  {% if context.query %}
```

Comment:
Do we know why this was like this in the first place?

Reply:
I resolved a merge conflict incorrectly.